config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  # Health check route
   get 'up' => 'rails/health#show', as: :rails_health_check
 
-  # Defines the root path route ("/")
-  # root "posts#index"
+  resources :users
+resources :comments
+resources :likes
+resources :posts
```

Comment:
- Screenshot 

![routes](https://github.com/AlyAtalla/blog-app/assets/107632780/795acfc9-5248-4a54-8d26-471d5761482c)


- [ ] You have done well implementing the routes, but for your routes to look like the one in the requirement screenshot, you need to wrap the `:posts` resources in the `users resources. 
- [ ] Also according to the requirement, you only currently need the `index and show` routes. :+1: :small_airplane:  Here a snippet of how your routes should be implemented:
```ruby

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end


```